company_country/models/res_config.py

```diff
@@ -1,21 +1,34 @@
 # Copyright 2016 Vauxoo
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import logging
 import os
 
 from odoo import _, api, models
 from odoo.exceptions import ValidationError
 
+_logger = logging.getLogger(__name__)
+
 
 class CompanyCountryConfigSettings(models.TransientModel):
     _name = 'company.country.config.settings'
     _description = 'Company Country Configuration Settings'
 
     @api.model
     def load_company_country(self, country_code=None):
+        account_installed = self.env['ir.module.module'].search([
+            ('name', '=', 'account'),
+            ('state', 'in', ('installed', 'to upgrade')),
```

Comment:
```suggestion
            ('state', 'in', ('installed')),
```

Reply:
@moylop260 
Are you sure?

Note that `to upgrade` indicates that the module is already installed, but it's about to be updated (which would be the case if updating all modules AFAIK).

Reply:
Done @moylop260